umbraco-cloud/databases/cloud-database.md

```diff
@@ -62,6 +62,6 @@ and in one of these folders for IIS Express:
 
 If you're using Visual Studio 2015+ check this path: `$(solutionDir)\.vs\config\applicationhost.config`
 
-In Visual Studio 2015+ you can also configure which `applicationhost.config` file is used by altering the `<UseGlobalApplicationHostFile>true|false</UseGlobalApplicationHostFile>` setting in the project file (eg: MyProject.csproj). (source: MSDN forum)
+In Visual Studio 2015+ you can also configure which `applicationhost.config` file is used by altering the `<UseGlobalApplicationHostFile>true|false</UseGlobalApplicationHostFile>` setting in the project file (eg: MyProject.csproj). (source: MicroSoft Developer Network (MSDN) forum)
```

Comment:
To avoid having so many brackets ()
```suggestion
In Visual Studio 2015+ you can also configure which `applicationhost.config` file is used by altering the `<UseGlobalApplicationHostFile>true|false</UseGlobalApplicationHostFile>` setting in the project file (eg: MyProject.csproj). Source: MicroSoft Developer Network (MSDN) forum.
```